memory-viz/src/tests/draw.spec.tsx

```diff
@@ -765,4 +765,79 @@ describe("draw function", () => {
         const svg: String = m.serializeSVG();
         expect(svg).toMatchSnapshot();
     });
+
+    it("logs a warning when provided 'small' width value", () => {
+        const objects: Array<Object> = [
+            {
+                type: "str",
+                id: 19,
+                value: "David is cool!",
+                style: ["highlight"],
+            },
+        ];
+        const spy = jest.spyOn(global.console, "log");
+        draw(objects, true, {
+            width: 13,
+            roughjs_config: { options: { seed: 12345 } },
+        });
+        expect(spy).toHaveBeenCalledTimes(1);
```

Comment:
Regarding your question about testing: the only part about testing the warning message that's tricky is the actual minimum width value. This suggests that you can take the actual warning message produced and check whether it matches a *regular expression* that has everything else fixed, and just accepts a number in place of the minimum width.